Gems/GradientSignal/Code/Source/Editor/EditorImageGradientComponentMode.cpp

```diff
@@ -28,6 +32,8 @@
 
 namespace GradientSignal
 {
+    static constexpr AZStd::string_view EditMenuIdentifier = "o3de.menu.editor.edit";
```

Comment:
Does this same string get hard-coded everywhere that something interacts with the edit menu?

Reply:
I have a follow-up PR ready to consolidate all these identifiers in headers, I had to wait for all migrations to be done before submitting it cause I was doing them in parallel :)